code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -99,12 +97,12 @@
 			add_attack_logs(user, M, "Injected with [name] containing [contained], transfered [trans] units", injected.harmless ? ATKLOG_ALMOSTALL : null)
 			to_chat(user, "<span class='notice'>[trans] units injected. [R.total_volume] units remaining.</span>")
 
+
```

Comment:
```suggestion
```
should just leave one space in between procs, unless there is a specific reason that you want to have 2 spaces here?

Reply:
nah just a missclick